lib/logstash/inputs/stomp.rb

```diff
@@ -37,6 +37,12 @@ class LogStash::Inputs::Stomp < LogStash::Inputs::Base
   #Auto reconnect interval in seconds
   config :reconnect_interval, :validate => :number, :default => 30
 
+  # Enable TLS/SSL connection?
+  config :ssl, :validate => :boolean, :default => false
+
+  # Validate TLS/SSL certificate?
+  config :verify_ssl, :validate => :boolean, :default => false
```

Comment:
Disabling SSL certificate verification only disables the checks that the certificate is specifically for the target host and that is signed by a good certificate authority.  The setting is needed to support self-signed certificates.

I agree that the default should probably be true.  I will make that change.